Gemfile.lock

```diff
@@ -524,4 +524,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   2.1.4
+   2.2.33
```

Comment:
Can we revert this line and squash into one commit 🦑

Reply:
ugh why does it change that when I only ask it to update a specific gem 😞

Reply:
yup sorted